x-pack/plugins/grokdebugger/public/sections/grokdebugger/components/custom_patterns_input/custom_patterns_input.js

```diff
@@ -33,11 +34,24 @@ app.directive('customPatternsInput', function () {
         $scope.aceLoaded = (editor) => {
           this.editor = editor;
           editor.getSession().setUseWrapMode(true);
```

Comment:
We duplicate this code several times across the three editors. It could be useful to create a function that applies these options/properties to the editor and takes min/max line numbers as params. We could put those line numbers in constants too.
@ycombinator WDYT?

Reply:
++ good idea!